src/popup/popup.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <section v-if="options && options.dashboards[0].url">
+    <div class="btn-group" v-if="options.dashboards.length > 1">
+      <button class="block" v-for="dashboard in options.dashboards" v-if="dashboard.url" @click="display(dashboard)">
```

Comment:
Buttons here need professional eye for styling ;)

Reply:
I choose to use Bulma for this project, so maybe you could have used [Tabs](https://bulma.io/documentation/components/tabs/)